akka-remote/src/main/scala/akka/remote/security/provider/DelegatingKeyManagerFactory.scala

```diff
@@ -0,0 +1,39 @@
+package akka.remote.security.provider
+
+import java.security._
+import java.util.Collections.{ emptyList, emptyMap }
+import javax.net.ssl.{ KeyManager, KeyManagerFactory, KeyManagerFactorySpi, ManagerFactoryParameters }
+
+private[akka] class DelegatingKeyManagerFactory extends KeyManagerFactorySpi {
```

Comment:
A bit bothersome, but this becomes `public` from Java, so we also need the annotation `@InternalApi` and a javadoc comment saying `/** INTERNAL API */` on all these